ecl/hqlcpp/hqlresource.cpp

```diff
@@ -5109,6 +5112,9 @@ void CSplitterInfo::gatherPotentialSplitters(IHqlExpression * expr, IHqlExpressi
     {
     case no_null:
     case no_fail:
+        return;
```

Comment:
If this separation is intended for a near future PR can we leave a comment here explaining this please?